sql/base_repository.py

```diff
@@ -0,0 +1,26 @@
+import mysql.connector
+
+
+class BaseRepository:
+    def __init__(self):
+        self.__connection = mysql.connector.connect(host="localhost", user="root", password="ua2023")
```

Comment:
pass all of this as parameters. because We should be able to reuse Base for any user.